judoscale-ruby/lib/judoscale/web_metrics_collector.rb

```diff
@@ -5,8 +5,10 @@
 
 module Judoscale
   class WebMetricsCollector < MetricsCollector
+    # NOTE: We collect metrics on all running web processes since they
+    # all receive and handle requests independently
     def self.collect?(config)
-      config.dyno.name == "web"
+      config.runtime_container.web?
```

Comment:
I think these helper methods ended up feeling cleaner than the `attr_reader`s (probably the question marks 😆)